### PR TITLE
Enter press on input applied. emptyEdit and labelPlaceHolder props added

### PR DESCRIPTION
**Pull request by cheekujha**
Changes
1. Enter key functionality applied.
2. **emptyEdit** prop - this prop keeps label in edit mode when text is empty(default **false**)
3. **labelPlaceHolder** prop - The default label value to be shown when text is empty.

### CODE REVIEW

**Review by bfischer — APPROVED** [review]

**Comment by mischah** [comment]
Would be awesome to get this in and released :octocat:

**Comment by vemuez** [comment]
So is this just waiting on a merge?

**Comment by NastuzziSamy** [comment]
Hey @cheekujha @mischah @vemuez  ! 

I uploaded a new package, look here:
https://github.com/bfischer/react-inline-editing/issues/11

**Comment by Yashkochar20** [comment]
Can we merge this ?

**Review by Yashkochar20 — APPROVED** [review]

**Comment by Yashkochar20** [comment]
@bfischer Lets get this in, it will be huge help.

**Comment by cheekujha** [comment]
Anything else needed from my side for merge?

**Review by bfischer — APPROVED** [review]

**Comment by Yashkochar20** [comment]
We need to rebuild the project and update package.json before merging. @bfischer

**Comment by Yashkochar20** [comment]
@cheekujha @bfischer Users won't be able to take advantage of this changes if we don't upgrade the package version and rebuild the project to reflect these changes.

**Comment by cheekujha** [comment]
@Yashkochar20 : sorry dude. I don't have access to create a new package.

**Comment by Yashkochar20** [comment]
@bfischer Do you plan to push new changes ? If not, can you comment so that I can use any other library for my project

**Comment by bfischer** [comment]
Yes I will, I'll try to get it published up later tonight

**Comment by Yashkochar20** [comment]
@bfischer Thank You, Sir. Waiting for the changes.

**Comment by bfischer** [comment]
@Yashkochar20 @cheekujha I committed some work to remove the componentWillReceiveProps call 
because it is being deprecated and I've replaced it with componentDidUpdate: https://github.com/bfischer/react-inline-editing/commit/0d162cdf36e5fb46190b37ff090d481280656bec

Functionally should be the same, but take a look in case there is a use case for you guys that I may have missed. I'm going to hold off on publishing until then.

**Comment by Yashkochar20** [comment]
@bfischer It looks good to me

**Comment by cheekujha** [comment]
@bfischer : looks fine.

**Pull request merged**
